static/js/script.js

```diff
@@ -19,30 +17,36 @@ Array.from(document.querySelectorAll('a')).forEach(a => {
 })
 
 /* animation flash info */
-const flashes = document.querySelector('.flashes');
+const flashes = document.querySelector('.flashes')
 if (flashes) {
     setTimeout(() => {
         flashes.classList.add('hidden')
     }, 3000)
 }
 
 /* display profile social network field on click */
-const socialIcon = document.querySelector('.social-networks li');
-socialIcon.addEventListener('click', (event) => {
-    event.preventDefault();
-    this.document.querySelector('label').classList.toggle('active')
-});
+const socialIcon = document.querySelector('.social-networks li')
+Array.from(document.socialIcon.forEach(li => {
```

Comment:
because you have used parenthesis everywhere in your script even for single or no parameter, you should also write `(li) => {` here